serve/handlers/tx/tx.go

```diff
@@ -26,23 +28,23 @@ func (h *Handler) GetTxHandler(
 	params []any,
 ) (any, *spec.BaseJSONError) {
 	// Check the params
-	if len(params) < 2 {
+	if len(params) != 2 {
```

Comment:
Is there a good way to do this? 
I was thinking that it would be good for the handler to tell the user that the number of parameters is incorrect.

Reply:
@zivkovicmilos ,
I modified the test case by changing the check condition.
Please check it again. 
https://github.com/gnolang/tx-indexer/pull/36/commits/f82a2dfb3bb87998d62664c89270b79e9647c063#diff-ad71bc5f39dad4d82341bdbffe8fb566d131bad2ea45a6f0afc8401e37a1f5beL26

In my opinion, the reason the tests in the `main` branch passed was because the error occurred in the type assertion, not the expected error.